corehq/apps/accounting/management/commands/get_partner_domain_user_history.py

```diff
@@ -0,0 +1,80 @@
+import datetime
+
+from dateutil import parser
+
+from django.core.management import BaseCommand
+
+from corehq.apps.accounting.models import DomainUserHistory
+from corehq.apps.es import FormES
+from corehq.apps.es.users import UserES
+from corehq.apps.users.models import Invitation
+from dimagi.utils.dates import add_months_to_date
+
+
+class Command(BaseCommand):
+    help = """Gets often requested statistics for a given list of partners'
+    domains: number of mobile users, submissions and estimated web users broken
+    down by month."""
+
+    def add_arguments(self, parser):
+        parser.add_argument('start_date')
+        parser.add_argument('end_date')
+        parser.add_argument('domains')
+
+    def handle(self, start_date, end_date, domains, **kwargs):
+        start_date = parser.parse(start_date).date()
+        end_date = parser.parse(end_date).date()
+        partner_domains = domains.split(',')
+
+        self.stdout.write('\nproject\tmonth\tmobile users\tsubmissions\tweb users')
+        for domain in partner_domains:
+            domain_user_histories = DomainUserHistory.objects.filter(
+                domain=domain,
+                record_date__gte=start_date,
+                record_date__lte=end_date
+            )
```

Comment:
Seems like ordered results would be nice for this report
```suggestion
            ).order_by('record_date')
```